### PR TITLE
Copy "git clone https..." to clipboard when clicked

### PR DESCRIPTION
**Pull request by AndrewKeepCoding**
## Description

Copy "git clone https://github.com/microsoft/WinUI-Gallery" to clipboard when clicked.

## Motivation and Context

This way is easier and less error prone.

## How Has This Been Tested?

Run the app and confirmed that the text is copied to the clipboard.

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by AndrewKeepCoding** [comment]
To keep consistent alignment with the bug request icon in the `SettingsCard` below, I didn't use the `CopyButton` control here and used a `FontIcon` as `SettingsCard.ActionIcon` thus there's no fancy copy animation the `CopyButton` control has.

File: WinUIGallery/SettingsPage.xaml [hunk 1]
```diff
@@ -111,11 +111,19 @@
...
                         Text="{x:Bind Version}" />
                     <toolkit:SettingsExpander.Items>
-                        <toolkit:SettingsCard Header="To clone this repository">
+                        <toolkit:SettingsCard
+                            x:Name="toCloneRepoCard"
+                            Click="toCloneRepoCard_Click"
+                            Header="To clone this repository"
+                            IsClickEnabled="True">
                             <TextBlock
+                                x:Name="gitCloneTextBlock"
```
Comment (marcelwgn) [inline WinUIGallery/SettingsPage.xaml:120]: I think we should probably use a monospaced font and a separate background for this text (though this text was already like this before this PR)

Comment (AndrewKeepCoding) [reply WinUIGallery/SettingsPage.xaml:120]: I changed the font to `Consolas` just like the other places (Iconography/Geometry) that has a copy button.

image

Comment (AndrewKeepCoding) [reply WinUIGallery/SettingsPage.xaml:120]: > a separate background for this text

I'm not sure what kind of background you mean. 🤔
Is there any section in the app that I can use as a reference?

Comment (marcelwgn) [reply WinUIGallery/SettingsPage.xaml:120]: Huh interesting, I thought that we had a specific background for source code but apparently we no longer have that.
Thank you for using Consolas btw!

**Review by marcelwgn — APPROVED** [review]

**Pull request merged**